test/RachisTests/DatabaseCluster/ClusterDatabaseMaintenance.cs

```diff
@@ -79,15 +81,14 @@ public class ClusterDatabaseMaintenance : ReplicationTestsBase
                         await session.SaveChangesAsync();
                     }
                 }
-                WaitForUserToContinueTheTest((DocumentStore)store);
 
                 var res = await store.Admin.Server.SendAsync(new AddDatabaseNodeOperation(databaseName));
                 Assert.Equal(1, res.Topology.Members.Count);
                 Assert.Equal(1, res.Topology.Promotables.Count);
 
                 await WaitForRaftIndexToBeAppliedInCluster(res.RaftCommandIndex, TimeSpan.FromSeconds(5));
                 await WaitForDocumentInClusterAsync<IndexMerging.User>(res.Topology, databaseName, "users/1", u => u.Name == "Karmel", TimeSpan.FromSeconds(10));
-
+                await Task.Delay(TimeSpan.FromSeconds(5)); // wait for the observer 
```

Comment:
This is a bad idea, you just increased the test times in 5 seconds.
Instead, let the observer tell you when it is done